readme_renderer/rst.py

```diff
@@ -25,26 +24,8 @@
 
 class ReadMeHTMLTranslator(HTMLTranslator):
 
-    def depart_image(self, node):
-        uri = node["uri"]
-        ext = os.path.splitext(uri)[1].lower()
-        # we need to swap RST's use of `object` with `img` tags
-        # see http://git.io/5me3dA
-        if ext == ".svg":
-            # preserve essential attributes
-            atts = {}
-            for attribute, value in node.attributes.items():
-                # we have no time for empty values
-                if value:
-                    if attribute == "uri":
-                        atts["src"] = value
-                    else:
-                        atts[attribute] = value
-
-            # toss off `object` tag
-            self.body.pop()
-            # add on `img` with attributes
-            self.body.append(self.starttag(node, "img", **atts))
+    # Overrides base class not to output `<object>` tag for SVG images.
+    object_image_types = {'.swf': 'application/x-shockwave-flash'}
```

Comment:
Let's just make this `object_image_types = {}`, we don't actually want to support _any_ `<object>` tags at all.

Reply:
Done.